examples/FromJim/spice.mzLib

```diff
@@ -0,0 +1,794 @@
+<mzSpecLib 1.0>
+MS:1003188|library name = spice
+MS:1003190|library version = 1.0
+MS:1001017|release date = (UTC) 5/25/2023 3:45:29 PM
+
+<Spectrum=1>
+MS:1003061|library spectrum name = JWH-250-5OH
+MS:1003062|library spectrum index = 795
+MS:1000894|retention time = 3.53813
+MS:1000864|chemical formula = 
+MS:1002894|InChiKey = 
+MS:1009100|CASNo = 
+MS:1000868|Smiles formula = 
+MS:1009101|compound class = 
+MS:1003059|number of peaks = 51
+<Analyte=1>
+MS:1000744|selected ion m/z = 352.19000
+MS:1000045|collision energy = 0
+MS:1000041|charge state = 1
+MS:1000008|ionization type=MS:1000073|electrosprary ionization
+MS:1000465|scan polarity=MS:1000130|positive scan
+<Peaks>
+51.0236 41921.14

```

Comment:
The delimiter character between peak values must a `\t`, not a single space.

>Each line of peak data MUST be tab-separated values of 2, 3, or more columns as follows:

An annotation may have a space in it and while one could tolerate spaces with a pull parser or an LR parser paradigm, both of these are more complicated to implement.

Reply:
Updated my code to address this issue.